web/src/pages/index.tsx

```diff
@@ -86,12 +83,6 @@ const Home = (props: Props): ReactElement => {
       } else if (userData === undefined && error !== undefined) {
```

Comment:
[Dust] Is the `else if` conditional necessary? Can we keep what is inside the code block?

Reply:
The tests still pass without it but I like that it is explicitly spelled out, it makes it easier to read.